File: download_html.py
import urllib.request

url = 'https://toddboswell.ehost.com/Cameo/index.html'
req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36'})
try:
    with urllib.request.urlopen(req) as response:
        html = response.read().decode('utf-8')
        with open('index.html', 'w', encoding='utf-8') as f:
            f.write(html)
        print("Success! Downloaded index.html")
except Exception as e:
    print(f"Error: {e}")

File: download_html.js
const https = require('https');
const fs = require('fs');

const url = 'https://toddboswell.ehost.com/Cameo/index.html';
const options = {
  headers: {
    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36'
  }
};

https.get(url, options, (res) => {
  let data = '';
  res.on('data', (chunk) => {
    data += chunk;
  });
  res.on('end', () => {
    fs.writeFileSync('index.html', data);
    console.log('Success! Downloaded index.html');
  });
}).on('error', (err) => {
  console.log('Error: ' + err.message);
});
